src/sentry/models/groupsearchview.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from django.db.models import UniqueConstraint
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import region_silo_model
+from sentry.db.models.base import DefaultFieldsModel
+from sentry.db.models.fields.foreignkey import FlexibleForeignKey
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+from sentry.models.savedsearch import SortOptions
+
+
+@region_silo_model
+class GroupSearchView(DefaultFieldsModel):
+    """
+    A model for a user's view of the issue stream
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+    name = models.TextField(max_length=128)
+    user_id = HybridCloudForeignKey("sentry.User", on_delete="CASCADE", null=True)
+    organization = FlexibleForeignKey("sentry.Organization", null=True)
```

Comment:
Can this ever actually be null? It seems like either there will be a view for a `user/org`, or just an org

Reply:
This is actually a mistake, neither `user_id` nor `organization_id` can be null, since we need to know which user created the view, and in which org they created it in. Apologies if this misled you into believing that `user_id` could be null too - there should never be a case (at least in the current product implementation) where a view's `user_id` can be null.

Reply:
Right, I thought we just had a potential future where there are org level default views as well? I guess the not null can be removed when/if that happens

Reply:
Ah I see what you were thinking now - I think if we implement that feature, we would have to add a `scope="user"|"org"` column, since we still want to know which user created a default org level view.